test/tuples.test.ts

```diff
@@ -450,4 +450,14 @@ describe("Tuples", () => {
     //    ^?
     type test3 = Expect<Equal<res3, 362880>>;
   });
+
+  it("Keys", () => {
+    type res0 = Call<Tuples.Keys, [3, 4, 5]>;
+    //   ^?
+    type test0 = Expect<Equal<res0, 0 | 1 | 2>>;
+
+    type res1 = Call<Tuples.Keys, number[]>;
+    //   ^?
+    type test2 = Expect<Equal<res1, number>>;
+  });
```

Comment:
Could you move this under the `Objects`, and make it support both tuples and objects? I think `Objects.Values` would make a lot of sense too, to keep the API similar to JS Object.keys and Object.values (our versions would return unions, which I think makes sense since these aren't sorted, and we already have Object.Entries that returns a union of entries)

Reply:
I'll give this a shot